include/arch/x86/linker.ld

```diff
@@ -137,6 +137,19 @@ SECTIONS
 
 #include <linker/common-ram.ld>
 
+#ifdef CONFIG_MMU
+       /*
+        * Page Tables are located here if MMU is enabled.
+	*/
+	SECTION_DATA_PROLOGUE(mmu_page_tables, (OPTIONAL),)
```

Comment:
yeah let's not do this unless placing this in its own section solves a specific problem